src/iotjs_magic_strings.h

```diff
@@ -328,6 +328,7 @@
 #endif
 #if ENABLE_MODULE_TIZEN
 #define IOTJS_MAGIC_STRING_TIZEN "tizen"
+#define IOTJS_MAGIC_STRING_EVENT_APP_CONTROL "AppControl"
```

Comment:
Please use `#define IOTJS_MAGIC_STRING_APP_CONTROL "AppControl"` to unify with others.